datatypes/src/operations/image/colorizer.rs

```diff
@@ -458,21 +509,31 @@ impl<'c> ColorMapper<'c> {
                 min_value,
                 max_value,
                 no_data_color,
-                default_color,
+                color_fields,
             } => {
                 let value: f64 = value.as_();
                 if f64::is_nan(value) {
                     *no_data_color
-                } else if value < *min_value || value > *max_value {
-                    *default_color
+                } else if value < *min_value {
+                    match color_fields {
+                        ColorFields::DefaultColor { default_color } => *default_color,
+                        ColorFields::OverUnder { under_color, .. } => *under_color,
+                    }
```

Comment:
If you would implement over_color and under_color on this enum, you could call it here and also use it in `over_color` of the colorizer.

Reply:
Changed with https://github.com/geo-engine/geoengine/pull/684/commits/cacee41b1ac1cdda64e8404e7a84816f18cc8868